index.js

```diff
@@ -196,7 +196,10 @@ Promise.resolve()
     if (argv.watch) {
       const watcher = chokidar.watch(
         input.concat(dependencies(results)),
-        { usePolling: argv.poll }
+        {
+          usePolling: argv.poll,
```

Comment:
`argv.poll || false` ?

But if I remember right yargs defaults to `false` anyways, should be ok then